lib/src/input/input.component.scss

```diff
@@ -0,0 +1,30 @@
+@import '../sass/colors';
+@import '../sass/functions.scss';
+@import '../sass/mixins.scss';
+
+.hc-input {
+    color: $text;
```

Comment:
Future PR maybe, just noting it while it's on my mind... would be sweet if there were a "form-controls" sass file containing things that are common across controls... like the styles for borders (default vs error state), font size, padding, etc. This could be a set of variables that each component pulls in, or perhaps standalone sass file that people could use even if they for some reason can't use the angular components but won't style guide approved form controls.

Reply:
That's a great idea.  I had to add the error messages css to the typography css file, but it didn't really seem like it belonged there.  So a forms.scss would be really useful.  If you remember, can you add that as an issue once this PR gets merged?